src/EditorFeatures/Core.Cocoa/NavigationCommandHandlers/FindBaseSymbolsCommandHandler.cs

```diff
@@ -86,12 +69,14 @@ protected override bool TryExecuteCommand(int caretPosition, Document document,
 
                     using (Logger.LogBlock(
                         FunctionId.CommandHandler_FindAllReference,
-                        KeyValueLogMessage.Create(LogType.UserAction, m => m ["type"] = "streaming"),
+                        KeyValueLogMessage.Create(LogType.UserAction, m => m["type"] = "streaming"),
                         context.CancellationToken))
                     {
                         try
                         {
+#pragma warning disable CA2007 // Consider calling ConfigureAwait on the awaited task
```

Comment:
📝 These are fine, but we should have one dedicated follow-up issue to address all CA2007 suppressions in this project.

Reply:
Yup: #48871 to review all suppressions.